clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/controllers/EcsClusterController.java

```diff
@@ -37,4 +39,10 @@ public EcsClusterController(EcsClusterProvider ecsClusterProvider) {
   public Collection<EcsCluster> getAllEcsClusters() {
     return ecsClusterProvider.getAllEcsClusters();
   }
+
+  @RequestMapping(value = {"/ecs/ecsDescribeClusters/{account}/{region}"})
```

Comment:
Can we keep the path to be the same as the method name? (for instance if you're using getAllEcsClustersDescription, then the path would be `/ecs/ecsClustersDescription/{account}/{region}`